infrastructure/server/oidc_clients_spar.tf

```diff
@@ -43,10 +43,10 @@ resource "aws_cognito_user_pool_client" "test_spar_oidc_client" {
   callback_urls                                 = [
     "http://localhost:3000/",
     "https://oidcdebugggersecure-c6af30-dev.apps.gold.devops.gov.bc.ca/",
-    "https://nr-spar-webapp-test-frontend.apps.silver.devops.gov.bc.ca/"
+    "https://nr-spar-test-frontend.apps.silver.devops.gov.bc.ca/"
```

Comment:
Hi Catherine, I didn't see this in ticket requirement, wondering, does SPAR request this recent change to remove `webapp` from domain?

Reply:
yeah, their deployment url for test and prod changed.